README.md

```diff
@@ -1,2 +1,55 @@
 # err-topic-a-day
-An errbot plugin to post a topic a day to a channel for discussion
+
+## Description
+This Bot (http://errbot.io) posts to slack  plugin to post a topic a day to a channel for discussion
```

Comment:
Let's rephrase this a bit:

"This is a plugin for errbot (http://errbot.io) to manage a "topic-a-day" channel for daily discussion of topics."